lightning/src/chain/channelmonitor.rs

```diff
@@ -2322,6 +2317,13 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 				},
 			}
 		}
+
+		// If the updates succeeded and we were in an already closed channel state, then there's no
+		// need to refuse any updates we expect to receive afer seeing a confirmed commitment.
```

Comment:
The only case where we might have a secondary `CLOSED_CHANNEL_UPDATED_ID` arises from post-force-close preimage forwarding (i.e `ChannelMonitorUpdateStep::PaymentPreimage`) ? This could be verified.